Source/Dafny/DafnyPipeline.csproj

```diff
@@ -139,6 +139,22 @@
     <Reference Include="BoogieParserHelper">
       <HintPath>..\..\..\boogie\Binaries\BoogieParserHelper.dll</HintPath>
     </Reference>
+    <Reference Include="Mono.Cecil, Version=0.11.1.0, Culture=neutral, PublicKeyToken=50cebf1cceb9d05e">
```

Comment:
Will the Cecil library be slurped down automatically? Do we need to update some build-from-source instructions somewhere?

Reply:
Yup, nuget restore will take care of it. But since you mention it I notice the installation instructions technically only mention doing that for Boogie and not Dafny itself. I'll add that.

Reply:
I manually verified this dependency is retrievable on a Windows machine using Visual Studio 2017 as well. I'm hoping to add a second Windows-based CodeBuild project to automatically verify that for future changes!